lib/npg_warehouse/loader/run_info.pm

```diff
@@ -1,26 +1,27 @@
-package npg_warehouse::loader::run_status;
+package npg_warehouse::loader::run_info;
 
 use Moose;
 use MooseX::StrictConstructor;
 use DBIx::Class::ResultClass::HashRefInflator;
 use Readonly;
+use List::MoreUtils qw(any);
 
 use npg_tracking::Schema;
 
 our $VERSION = '0';
 
-Readonly::Array  our @RUN_STATUS_TABLES   => qw/ RunStatusDict RunStatus /;
+Readonly::Array  our @RUN_TABLES   => qw/ RunStatusDict RunStatus Run/;
```

Comment:
If the `Run` class in first in this list, in future we will be able to set a foreign key on the `id_run` column of the `iseq_run_status` table in the same way it is done in the tables we are copying from